### PR TITLE
Simple compatibility on Windows

### PR DESCRIPTION
**Pull request by KeziahMoselle**
Hello,

This looked great but i'm on Windows so I cannot launch it and try to make it run.

This is my first PR, please be gentle.

- [x] White icons
- [x] Exported the white icons for Windows
- [x] Fix error on the main process (app.dock.hide();)

### CODE REVIEW

**Comment by jamieweavis** [comment]
Thanks for the pull request @KeziahMoselle! I'm on holiday at the moment but I'll look at getting this merged when I'm back in a few days 👍

**Comment by KeziahMoselle** [comment]
After `npx build` on Windows and installed Streaker, Streaker don't launch automatically at login.
I have no clue how to fix that.

Edit : And this happen : preview

**Comment by KeziahMoselle** [comment]
Okay, probably need to change the path here : 
```js
const streakerAutoLauncher = new AutoLaunch({
    name: pjson.name,
    path: `/Applications/${pjson.name}.app`
  });
```
I will try to fix that soon.

**Comment by KeziahMoselle** [comment]
The weird [] was the tabulation \t at :
```js
{ label: `Current Streak:\t${currentStreak}`, enabled: false },
{ label: `Best Streak:\t\t${bestStreak}`, enabled: false },
{ label: `Contributions:\t${contributionCount}`, enabled: false },
```
When I removed them, the [] disappear : 
preview

File: src/app.js [hunk 1]
```diff
@@ -163,7 +163,9 @@ app.on('ready', () => {
     }
   }
 
-  app.dock.hide();
+  if (process.platform == 'darwin') {
```
Comment (jamieweavis) [inline src/app.js]: Could we please use strict equality checks? e.g. `process.platform === 'darwin'`

File: src/icon.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
 const path = require('path');
 
-module.exports = {
-  done: path.join(__dirname, 'icons/macos/doneTemplate.png'),
-  todo: path.join(__dirname, 'icons/macos/todoTemplate.png'),
-  load: path.join(__dirname, 'icons/macos/loadTemplate.png'),
-  fail: path.join(__dirname, 'icons/macos/failTemplate.png')
-};
+if (process.platform == 'win32') {
```
Comment (jamieweavis) [inline src/icon.js]: Again, strict equality checks please! Your editor/IDE should be complaining about this if it has eslint integration.

File: src/icon.js [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+  module.exports = {
+    done: path.join(__dirname, 'icons/win/doneTemplate.png'),
```
Comment (jamieweavis) [inline src/icon.js]: Icons only need to be suffixed with `Template` on macOS, this is for light and dark mode support - for Windows we can remove them.

I noticed that the tray icons look quite pixelated in your screenshot, according to the [Electron docs](https://electronjs.org/docs/api/tray#class-tray):
> On Windows it is recommended to use ICO icons to get best visual effects.

**Review by jamieweavis — CHANGES_REQUESTED** [review]

**Review by jamieweavis — APPROVED** [review]

**Review by jamieweavis — CHANGES_REQUESTED** [review]
Sorry for the delay, I've been quite busy lately.

Thanks for sorting out the icon, looking much better now!

Regarding the tabulation, I think we can safely remove it since it feels a little hacky and I'm not sure it adds any value anyway - in the future we could perhaps look at having the numbers aligned all the way to the right of the menu but that's out of scope of this issue.

As you found out, the autolauncher path will need updating depending on which platform we're on. The [auto-launch](https://github.com/Teamwork/node-auto-launch) package that we are using already supports Windows so it should just be a case of using the correct path when on Windows - it looks like the Windows installer installs to the path: `C:\Users\me\AppData\Local\Programs\Streaker`.

Once these two points have been addressed then this should be ready to merge!

**Comment by KeziahMoselle** [comment]
Yeah look's better !

I removed the \t and remove the path option, according to the docs : 
> For NW.js and Electron apps, you don't have to specify the path. We guess based on `process.execPath`.

And it works !

**Comment by KeziahMoselle** [comment]
I tested it on my MacBook and it works too, without the `path` field

**Review by jamieweavis — APPROVED** [review]

**Comment by jamieweavis** [comment]
Fantastic, thanks for this!

**Pull request merged**